ChangeLog.txt

```diff
@@ -1,13 +1,14 @@
-v2.x - 2017-xx-xx
+v1.3.x - 2017-xx-xx
```

Comment:
The next version is not yet defined to be 1.3 or 2.0.
That why I would not yet update the version.